README.md

```diff
@@ -55,13 +57,30 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
+
+`NOTE: Commands are to be executed on Linux, Mac, and Windows(using Powershell)`
+
+1. Download the Mattermost docs repo onto a machine with Python installed or simple cloned the forked copy of this repo by running,
+```sh
+$ git clone https://github.com/Your_Username/docs.git
```

Comment:
What would be the difference between cloning the docs repo and cloning your fork? The two URLs are different so if both options are provided it might be a consideration to provide the docs repo URL (https://github.com/mattermost/docs.git).

Reply:
Sure, updating to as recomended.
Thanks